server_installation/topics/operator/installation.adoc

```diff
@@ -0,0 +1,145 @@
+
+[[_installing-operator]]
+=== Installing the {project_operator} on a cluster
+
+To install the {project_operator}, you can use:
+
+* <<_install_by_olm,The Operator Lifecycle Manager (OLM)>> 
+* <<_install_by_command,Command line installation>> 
+
+[[_install_by_olm]]
+==== Installing using Operator Lifecycle Manager
+
+ifeval::[{project_community}==true]
+You can install the Operator on a OpenShift or Kubernetes cluster
+
+===== Installation on an OpenShift cluster
+endif::[]
+
+.Prerequisites
+
+You have cluster administrator permissions or an equivalent level of permissions granted by an administrator.
+
+.Procedure
+
+Perform this procedure on an OpenShift 4.3 cluster.
+
+. Open the OpenShift Container Platform web console.
+
+. In the left column, click `Operators, OperatorHub`.
+
+. Search for {project_name} Operator.
++
+.OperatorHub tab in OpenShift
+image:{project_images}/operator-openshift-operatorhub.png[]
+
+. Click the {project_name} Operator icon.
++
+An Install page opens.
++
+.Operator Install page on OpenShift
+image:{project_images}/operator-olm-installation.png[]
+
+. Click `Install`.
+
+. Select a namespace and click Subscribe.
++
+.Namespace selection in OpenShift
+image:images/installed-namespace.png[]
++
+The Operator starts installing.
+
+When the Operator installation completes, you are ready to create custom resources to install {project_name}, create realms, and perform other adminstrative tasks.
+
+ifeval::[{project_community}==true]
+
+===== Installation on a Kubernetes cluster
+
+.Prerequisites
+
+You have cluster administrator permissions or an equivalent level of permissions granted by an administrator.
+
+.Procedure
+
+For a Kubernetes cluster, perform these steps.
```

Comment:
I am just curious, which version of Kubernetes is this one, because I am not sure that Kubernetes has a GUI where you can modify something, the dashboard is more like read only view not like Openshift GUI.

Reply:
@andymunro I do not know is you observed this comment. So I am just assigning it to you. Thank you for your help. :-)